coralproject/talk
# [CORL-2860] update unanswered tags when rejecting comments

## What does this PR do?

Update (remove) the unanswered tags when we reject comments so that Q&A tab counts update appropriately when moderation actions are performed.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indices.

## How do I test this PR?

- open up a Q&A stream
- post a question
- use a mod account to reject the unanswered question
- refresh / switch tabs and see that the `Unanswered` tab count updates correctly

## Where any tests migrated to React Testing Library?

No

## How do we deploy this PR?

No special considerations.
